xml/Microsoft.Vsa/IVsaIDESite.xml

```diff
@@ -48,8 +48,8 @@
         <Parameter Name="optional" Type="System.Object" />
       </Parameters>
       <Docs>
```

Comment:
Same as earlier. Let me see if looking at the code can help me come up with some descriptions for these two params.